src/components/toolbar/Toolbar.tsx

```diff
@@ -55,45 +44,15 @@ export function Toolbar(props: ToolbarProps) {
     [intent, large, vertical, disabled],
   );
 
-  const ref = useRef<HTMLDivElement>(null);
-
-  // Work around wrong width on vertical flex when wrapping
-  // In Chrome: recently fixed (https://bugs.chromium.org/p/chromium/issues/detail?id=507397)
-  // In Firefox: work-around needed (https://bugzilla.mozilla.org/show_bug.cgi?id=995020)
-  // In Safari: work-around needed
-  useLayoutEffect(() => {
-    if (!vertical) {
-      return;
-    }
-
-    function update() {
-      const lastElement = ref.current?.lastElementChild;
-      if (!lastElement) {
-        return;
-      }
-      ref.current.style.width = 'initial';
-      const divRect = ref.current.getBoundingClientRect();
-      const lastElemRect = lastElement.getBoundingClientRect();
-      const width = `${lastElemRect.right - divRect.left}px`;
-      if (ref.current.style.width !== width) {
-        ref.current.style.width = width;
-      }
-    }
-
-    const element = ref.current;
-    if (element) {
-      const observer = new ResizeObserver(update);
-      observer.observe(element);
-      return () => observer.unobserve(element);
-    }
-  }, [vertical]);
```

Comment:
I would be surprised if this workaround is no longer needed.

Please check the vertical layout with wrapping in the stories with Chrome, Firefox and (if possible) Safari.

Reply:
Okay

Reply:
It doesn't work everytime so better keep it